src/profile.c

```diff
@@ -361,23 +357,30 @@ static inline UInt getFilenameId(Stat stat)
   return id;
 }
 
+static inline Int8 CPUmicroseconds()
+{
+#if defined(HAVE_GETTIMEOFDAY)
+  struct timeval timebuf;
```

Comment:
Just to make sure you didn't miss this. I'd like to merge, but I think `HAVE_GETRUSAGE` is the correct constant to `\#ifdef` here?